docs/_templates/autosummary_core/pydantic_model.rst

```diff
@@ -0,0 +1,11 @@
+{% if referencefile %}
```

Comment:
I'm just curious, how did you know that a `pydantic_model.rst` template would get used as the template by the autodoc_pydantic documenter? I can't follow from the docs I've seen how this works.

Reply:
Alas, the answer is "spending a bunch of time reading the source code while thinking about making a PR to improve support." I figured it out as part of my research for SQR-077.